### ISSUE TITLE
[Techstack-beta] Nixpkgs and NixOS manuals regressed to default styles

### ISSUE THREAD

**Issue by samueldr** [NixOS/nixos-homepage#1251]
This also applies to the pills.

![image](https://github.com/NixOS/nixos-homepage/assets/132835/333c8d06-00a2-41fc-bcba-ffb616ba6aea)

**Comment by thilobillerbeck** [comment]
Deciding on how to embed the docs should, as the rest of the documentation, be a task of the docs team. But maybe @garbas kan tell you more about that.

**Comment by samueldr** [comment]
I'm sorry to dump this here, but this is not aimed at you, @thilobillerbeck.

This again shows that the execution of handling the technicalities of building the website shouldn't be handled by the marketing team. The marketing team should handle marketing. (And the marketing team shouldn't handle documentation. Which was a problem when I was still involved.)

Similarly, the documentation team should author documentation. Not deal with having the form of the documentation be coherent with the site.

This was a working feature.

This is a regression.

**Comment by asymmetric** [comment]
Should this issue be open? The issue is still present on the website.

**Comment by garbas** [comment]
The unthemed pills and manual is intended behavior. They will be also.managed by the docs team (hopefully soon) where theme can be applied.

**Comment by infinisil** [comment]
As an active docs team member this is the first time I hear about that..

Ping @NixOS/documentation-team, did anybody know about this?

**Comment by fricklerhandwerk** [comment]
At some point we agreed to move the manuals to nix.dev, yes. I still intend to make it happen one way or another. I started in October and didn't manage to finish yet because of conflicting priorities due to too many responsibilities.

IMO it's okay, we have more pressing issues than style sheets.

**Comment by 9999years** [comment]
...Huh. I'd noticed the "default" stylesheet but assumed it was just a new stylesheet. The reduced whitespace, particularly around the table of contents, makes it much easier to parse. I thought it was an improvement, even if the typefaces didn't match the nixos.org homepage and whatnot.

Agree that the intended stylesheets should be applied, but we should make a note to go fix the whitespace in the table of contents. (Where are those stylesheets? I could open a PR.)

**Comment by DanielSidhion** [comment]
The default stylesheet is "incomplete", in the sense that certain elements (such as caution blocks) are unstyled and cause confusion when reading the manual.

The way that things worked before is that the manuals were built with the default stylesheets, and the previous homepage added extra stylesheets on top of that to get to the style we know.

With the move to astro, those extra stylesheets were removed and not ported anywhere, so we get only the default ones. I'm working to bring those extra stylesheets into the default, which should make the manual look the way we expect in the astro homepage, and also will have the benefit of showing documentation writers the final state of the manual during development (we just accepted that the manual we saw during development was different).
